src/test/java/org/opendatakit/common/persistence/QueryResultTest.java

```diff
@@ -14,6 +14,9 @@
 package org.opendatakit.common.persistence;
 
 import static org.junit.Assert.assertEquals;
+import static org.opendatakit.common.persistence.Query.Direction.*;

```

Comment:
I am not a big fan of star import :)

Reply:
I'm curious about this... why? IntelliJ by default does this for you whenever you import more than 3 or 5 members from the same class...

Reply:
I like star import in certain cases, such as with an enum when you’re using all the values.

Reply:
Honestly I didn't think about this before. I don't really care because IntelliJ will handle it. I suggest we let checkstyle make our lives miserable on a future PR :P